### ISSUE TITLE
this.getValueFromSource(...).then is not a function

### ISSUE THREAD

**Issue by tobias-g** [jacob-meacham/serverless-plugin-git-variables#4]
Hi getting the following error.

```
  Type Error ---------------------------------------------
 
     this.getValueFromSource(...).then is not a function
 
     For debugging logs, run again after setting the "SLS_DEBUG=*" environment variable.
 
  Get Support --------------------------------------------
     Docs:          docs.serverless.com
     Bugs:          github.com/serverless/serverless/issues
     Forums:        forum.serverless.com
     Chat:          gitter.im/serverless/serverless
 
  Your Environment Information -----------------------------
     OS:                 darwin
     Node Version:       6.10.3
     Serverless Version: 1.13.2
```

Looks like `_getValueFromGitSync` is expected to return a promise but isn't.

**Comment by jacob-meacham** [comment]
Hi tobias-g - Thanks for the ticket! I'll look into it this weekend. As you probably know, serverless itself is changing very quickly and it's tough to keep plugins working across all the different versions in the wild.

**Comment by Christoph-Schabert** [comment]
I have the same problem in 1.16

**Comment by marta-tatiana** [comment]
same here, 1.17.0

**Comment by jacob-meacham** [comment]
Thanks to @marta-tatiana this is now fixed for serverless 1.16.x and above!
